get-to-know-syskit-point/navigate-through-syskit-point.md

```diff
@@ -8,33 +8,48 @@ description: This article describes how to navigate Syskit Point when starting t
 
 Here, you have to enter your existing Microsoft 365 credentials.
 
 {% hint style="warning" %}
 **Please note!**  
-When you are signing in for the first time, it is advisable to use Global Administrator credentials and proceed with the [Role-based Access configuration](../configuration/enable-role-based-access.md).
+When signing in for the first time, using Global Administrator credentials and proceeding with [Role-based Access configuration](../configuration/enable-role-based-access.md) is advisable.
 {% endhint %}
 
 After the sign-in, the Syskit Point Home screen opens.
 
 ## Home Screen
 
-Place where everything in Syskit Point starts. You can see 4 major sections:
+The home screen is where everything in Syskit Point starts. 
+
+You can see 8 major sections:
+
+ * **Inventory** - shows the number of each type of workspace in your tenant; clicking View All opens the Sites Overview screen  
+ * **Users** - shows the number of users in your environment; clicking View All opens the Users Overview screen
+ * **Storage** - shows the amount of your used tenant's storage, as well as which workspaces utilize the storage; clicking View All opens the Storage Metrics report
+ * **Security & Compliance** - shows the vulnerabilities in your environment through security and compliance checks; clicking View All opens the Security & Compliance checks dashboard
+ * **External Collaboration** - shows a list and number for all external collaboration in your environment
+ * **Active Access Review** - shows the progress of your active Access Review tasks; clicking View All opens the Govern Overview screen on the Scheduled Review section of the Access Review 
+ * **Lifecycle Management** - shows the progress of your Lifecycle Management tasks; clicking View All opens the Govern Overview screen on the Overview section of Lifecycle Management
+ * **Microsoft 365 Licenses** - shows an overview of your Microsoft 365 Licenses; clicking View All opens the Licenses Overview report
+
+On the left side of the screen, you will find buttons to access the following features:
 
 * **Sites** – complete overview and inventory of all sites and ownerships. Full list of classic sites, sites connected to MS Teams and Microsoft 365 Groups, and OneDrive as well. 
 * **Microsoft Teams & Microsoft 365 Groups** – a complete overview of all Teams and Microsoft 365 Groups detected in your environment. 
-* **Users** – explore all users and see more details about their memberships and permissions they hold across the sites. 
-* **Reports** – create security and audit activities across your workspaces. 
+* **Users** – explore all users and see more details about their memberships and the permissions they hold across the sites. 
+* **Report** – create security and audit activities across your workspaces. 
+* **Tasks** - resolve current tasks and keep your environment secure and organized
+* **Govern** - stay up to date with governance in your environment
 
```

Comment:
tu bi dodao i Settings s napomenom da je to dostupno samo Syskit Point Administratorima